### PR TITLE
remove 1.14 from comments and others

### PR DESCRIPTION
**Pull request by J3fftw1**
## Description

This pr removes 1.14 and 1.15 from readme, bug report form, pull request form and other comments.
Full credits to @variananora 
Forgot to make a different branch :notLikeThis:
## Proposed changes

updated readme, bug report form, pull request form and other comments.
## Checklist

- [x] I have fully tested the proposed changes and promise that they will not break everything into chaos.
- [x] I have also tested the proposed changes in combination with various popular addons and can confirm my changes do not break them.
- [x] I have made sure that the proposed changes do not break compatibility across the supported Minecraft versions (1.14.* - 1.19.*).
- [x] I followed the existing code standards and didn't mess up the formatting.
- [x] I did my best to add documentation to any public classes or methods I added.
- [ ] I have added `Nonnull` and `Nullable` annotations to my methods to indicate their behaviour for null values
- [ ] I added sufficient Unit Tests to cover my code.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Here is a full summary of the differences between the two different versions of
 
 | | development (latest) | "stable" |
 | ------------------ | -------- | -------- |
-| **Minecraft version(s)** | :video_game: **1.14.\* - 1.19.\*** | :video_game: **1.14.\* - 1.19.\*** |
+| **Minecraft version(s)** | :video_game: **1.16.\* - 1.19.\*** | :video_game: **1.14.\* - 1.19.\*** |
 | **Java version** | :computer: **Java 16 (or higher)** | :computer: **Java 8 (or higher)** |
```
Comment (variananora) [inline README.md:34]: RC-33 is now J16
```suggestion
| **Minecraft version(s)** | :video_game: **1.16.\* - 1.19.\*** | :video_game: **1.14.\* - 1.19.\*** |
| **Java version** | :computer: **Java 16 (or higher)** | :computer: **Java 16 (or higher)** |
```

**Review by variananora — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -40,7 +40,7 @@
 #### Additions
 
 #### Changes
-
+* Removed 1.14.* and 1.15.* support
 #### Fixes
```
Comment (variananora) [inline CHANGELOG.md:44]: you need spaces here, idk why the suggestion doesnt keep the new line

```suggestion
* Removed 1.14.* and 1.15.* support

#### Fixes
```

**Review by variananora — APPROVED** [review]
LGTM

**Review by TheBusyBiscuit — APPROVED** [review]
LGTM thanks

**Pull request merged**